### PR TITLE
[1241] Add support email link to footer

### PR DESCRIPTION
**Pull request by dankmitchell**
### Changes proposed in this pull request
Add support email link to footer

### Guidance to review

# After
Screenshot 2019-04-03 at 20 18 25

### CODE REVIEW

**Comment by benilovj** [comment]
Just noticed a very subtle text colour change between mcui and mcf - dunno if it's deliberate or not:

**mcf**
image

**mcui**
image

**Comment by dankmitchell** [comment]
@benilovj it was deliberate as I didn't see any benefit in adding the CSS rule into the rails app to make the text black. Happy to include. CC. @fofr

**Comment by fofr** [comment]
The new one is better.

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -85,6 +85,13 @@
 
     <footer class="govuk-footer " role="contentinfo">
       <div class="govuk-width-container ">
+        <div class="govuk-grid-row">
+          <div class="govuk-grid-column-two-thirds govuk-footer__content-container">
+            <p class="govuk-footer__content">
+              <%= mail_to "becomingateacher@digital.education.gov.uk", "Contact the Becoming a Teacher team", subject: "Publish teacher training courses feedback", class: "govuk-footer__link" %> for support.
```
Comment (fofr) [inline app/views/layouts/application.html.erb]: Subject should probably say "Publish teacher training courses support"

**Pull request merged**